tools/make-docs.pl6

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env perl6
+
+use File::Find;
+
+sub MAIN (:$filename, :$output = 'API.pm') {
+
+  my @files = $filename ??
+      $filename.Array
+      !!
+      find
+        dir => 'lib',
+        name => /'.pm6' $/;
+
+  my $apiDocsDir = $*CWD;
+  unless $apiDocsDir.add('lib').e {
+    say 'Please run this script from the project root directory.';
+    exit;
+  }
+
+  $apiDocsDir .= add($_) for <docs api>;
+  $apiDocsDir.mkdir;
+
+  my @destFiles;
+  for @files {
+    my $newFile = $apiDocsDir;
+
+    for $*SPEC.splitdir( .relative ).skip(1) {
+      $newFile .= add($_);
+      $newFile.mkdir unless $newFile.Str.ends-with('.pm6');
```

Comment:
Don't we want to render `.md` files instead of `.pm6`?

Reply:
Oops! Yes! Yes, we do.